Makefile

```diff
@@ -178,6 +178,15 @@ TOUCHID_MESSAGE := with-Touch-ID
 TOUCHID_TAG := touchid
 endif
 
+# Enable VNet daemon?
+# With VNETDAEMON=yes, tsh uses a Launch Daemon to start VNet.
+# This requires a signed and bundled tsh.
+VNETDAEMON_MESSAGE := without-VNet-daemon
+ifeq ("$(VNETDAEMON)", "yes")
+VNETDAEMON_MESSAGE := with-VNet-daemon
+VNETDAEMON_TAG := vnetdaemon
```

Comment:
I have [a branch with updates to teleport.e](https://github.com/gravitational/teleport.e/tree/r7s/vnet-1-plist) ready to go as well. I wanted to wait with creating a PR until I get an approval here, as GitHub is going to assign code owners for a review, even if I create the PR as a draft.